api/v1beta1/keystoneservice.go

```diff
@@ -21,107 +21,26 @@ import (
 	"time"
```

Comment:
this is just part of a split out of the func.go into keystoneapi.go, keystoneendpoint.go and keystoneservice.go